javascript-loops-practice/countdown.js

```diff
@@ -0,0 +1,9 @@
+/* exported countdown */
+function countdown(number) {
+  var array = [];
+  var i = number;
+  for (i; number >= 0; number--) {
```

Comment:
Why do this separately? You should just do this in the initialization

Reply:
doing more work than i need to lol